backend/satellite_tools/mgr-sign-metadata

```diff
@@ -26,11 +27,11 @@ if [ -z "$GPGPASS" ]; then
     exit 1
```

Comment:
This check also fails when the passphrase is empty. Shall we not allow keys with no passphrases?

@mcalmer what do you think?

Reply:
I think we can ignore this. I am even unsure if GPG allows you to not define a password:-)

Reply:
I tested and `gpg` allows empty passwords. However the `mgr-sign-metadata` script doesn't allow using an empty password. I'll change the empty pass warning in the `mgr-sign-metadata-ctl` to an err.

Reply:
Changed `mgr-sign-metadata-ctl` to complain with an error if the `GPGPASS` is empty.